packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -555,8 +568,19 @@ const Summary: React.FunctionComponent = () => {
 				} )
 			) }
 
-			{ /* @TODO: placeholder for https://github.com/Automattic/wp-calypso/issues/47392 */ }
-			<Link to={ Route.Success }>{ __( 'Launch your site', __i18n_text_domain__ ) }</Link>
+			<div className="focused-launch-summary__actions-wrapper">
+				<ActionButtons className="focused-launch-summary__launch-action-bar">
+					{ /* @TODO: placeholder for https://github.com/Automattic/wp-calypso/issues/47392 */ }
+					<Link to={ Route.Success }>{ __( 'Launch your site', __i18n_text_domain__ ) }</Link>
```

Comment:
Shouldn't this be an `<NextButton/>` instead of a `<Link/>`?

Reply:
See reply in https://github.com/Automattic/wp-calypso/pull/47624#issuecomment-732234869